src/components/Questionnaire/QuestionTypes/ConsentQuestion.tsx

```
@@ -0,0 +1,405 @@
+  CONSENT_STATUSES,
+  CreateConsentRequest,
+  VERIFICATION_TYPES,
+} from "@/types/consent/consent";
+import { FileUploadQuestion } from "@/types/files/files";
+import { QuestionValidationError } from "@/types/questionnaire/batch";
+import {
+  QuestionnaireResponse,
+  ResponseValue,
+} from "@/types/questionnaire/form";
+import { Question } from "@/types/questionnaire/question";
+import { validateFields } from "@/types/questionnaire/validation";
```

ESLint

[error] 'validateFields' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)